test/parser/samples/error-binding-mustaches/error.json

```diff
@@ -1,5 +1,5 @@
 {
-	"message": "bound values should not be wrapped — use 'foo', not '{{foo}}'",
+	"message": "directive values should not be wrapped — use 'foo', not '{{foo}}'",
```

Comment:
I'm not sure this tests the same thing. A directive is *not* the same thing as a bound value...

Reply:
Directives are special attributes with a colon. Bindings happen to be one type of directive, but all directive expressions should not be wrapped. This is a more general message for any directives. I could change it to use the specific name of the directive. E.g. `Binding values should not be wrapped` and `EventHandler values should not be wrapped`. But since the error points to the code where the issue is, I don't feel this provides much.

Reply:
Can see both sides of this, though I think it's probably ok to leave in —

```
on:click='{{set({foo: 'bar' })}}'
```

is just as incorrect as `bind:value='{{foo}}'`, it's just less likely that someone would make that mistake.